dolphinscheduler-bom/pom.xml

```
@@ -535,7 +536,17 @@
                     </exclusion>
                 </exclusions>
             </dependency>
-
+            <dependency>
+                <groupId>org.apache.kyuubi</groupId>
+                <artifactId>kyuubi-hive-jdbc</artifactId>
```

The shaded JDBC client is recommended
```suggestion
                <artifactId>kyuubi-hive-jdbc-shaded</artifactId>
```

---

got it.